sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/implementation/Utility.java

```diff
@@ -0,0 +1,460 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
We have a utilities-ish class called EntityHelper, could we move it there instead of having another utility class?

Reply:
moved to EntityHelper class.